content/help/ops/component-debugging/index.md

```diff
@@ -10,24 +10,42 @@ how to get under the hood.
 
 ## With `istioctl`
 
-`istioctl` allows you to inspect the current xDS of a given Envoy from its admin interface (locally) or from Pilot using the `proxy-config` or `pc` command.
+### Get an overview of your mesh
 
-For example, to retrieve the configured clusters in an Envoy via the admin interface run the following command:
+You can get an overview of your mesh via `proxy-status` command:
 
 {{< text bash >}}
-$ istioctl proxy-config endpoint <pod-name> clusters
+$ istioctl proxy-status
 {{< /text >}}
 
-To retrieve endpoints for a given pod in the application namespace from Pilot, run the following command:
+If a proxy is missing from the output list it means that it is not currently connected to a Pilot instance so will not be receiving any configuration.
```

Comment:
Also if it is stale that likely means there are networking issue or Pilot needs to be scaled.